### PR TITLE
feat(function/exports): AddSharedAccount(society, amount)

### PR DESCRIPTION
**Pull request by wasabirobby**
Gives the ability to create shared accounts during runtime as well as exports to integrate with 3rd party addons

### CODE REVIEW

**Review by Gellipapa — COMMENTED** [review]
Please handle nil values.

**Review by Thekuca — CHANGES_REQUESTED** [review]
please change to oxmysql funcs and don't use named placeholders (slower) for sql tasks

**Comment by Gellipapa** [comment]
@wasabirobby Hi! Do you can fix request changes and resolved merge conflict? Please.

**Review by wasabirobby — COMMENTED** [review]
I hope recent commit solves comments

File: server/main.lua [hunk 1]
```diff
@@ -44,6 +44,27 @@ function GetSharedAccount(name)
 	return SharedAccounts[name]
 end
 
+function AddSharedAccount(society, amount)
```
[the remark this answers is not in the document]
Comment (wasabirobby) [reply server/main.lua:47]: Done :)

**Review by Gellipapa — APPROVED** [review]

**Comment by Gellipapa** [comment]
@wasabirobby Thank you fixes.

**Pull request merged**